pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -446,45 +472,57 @@ protected void handleConnect(CommandConnect connect) {
     @Override
     protected void handleSubscribe(final CommandSubscribe subscribe) {
         checkArgument(state == State.Connected);
-        final String topicName = subscribe.getTopic();
         final long requestId = subscribe.getRequestId();
         final long consumerId = subscribe.getConsumerId();
+
+        DestinationName topicName;
+        try {
+            topicName = DestinationName.get(subscribe.getTopic());
```

Comment:
it's small code but it seems duplicate for all handler. so, should we move it to common method?